check_source.py

```diff
@@ -209,6 +209,13 @@ def check_action_add_role(self, request, action):
         return False
 
     def check_action_delete(self, request, action):
+        if action.tgt_repository is not None:
+            if action.tgt_project.startswith('openSUSE:'):
+                self.review_messages['declined'] = 'removing repository'
```

Comment:
"""The repositories in the openSUSE:* namespace are managed by the Release Managers. For suggestions on changes, please send a mail to opensuse-releaseteam@opensuse.org with an explanation why you think that change makes sense""" ?

After all this is a message we give to the user as an explanation why his request is declined. "removing repository" as a decline message sounds like it would still be done.